migrator/src/main/java/uk/nhs/digital/ps/migrator/config/ExecutionConfigurator.java

```diff
@@ -155,20 +143,20 @@ private Path getPathArg(final ApplicationArguments args, final String commandLin
                     " NOTE that the directory is deleted and re-created on each run."
             ),
             describe(
-                TAXONOMY_SPREADSHEET_PATH,
-                "Path to the spreadsheet that contains the taxonomy we want to import into hippo." +
-                    " Optional - if not provided, taxonomy will not be imported."
+                TAXONOMY_DEFINITION_IMPORT_PATH,
+                "Path to the spreadsheet that contains the taxonomy definition we want to import into hippo." +
+                    " Required if --" + NESSTAR_CONVERT_FLAG + " is specified, optional otherwise."
```

Comment:
Is this arg now mandatory?

Reply:
Yes in the nesstar convert task the following are asserted:
`assertRequiredArgs(hippoImportDir, nesstarUnzippedArchiveLocation, taxonomyDefinitionImportPath, taxonomyMappingImportPath);`